support-workers/riff-raff.yaml

```diff
@@ -26,3 +26,4 @@ deployments:
     parameters:
       bucket: support-workers-dist
       cacheControl: private
+      publicReadAcl: false
```

Comment:
this is ok because it's granted explicit access
https://github.com/guardian/support-frontend/blob/4afd67abb4a8adb11ce974db31dd36ff180df7fe/support-lambdas/it-test-runner/cfn.yaml#L59

and uses the S3 client to fetch it
https://github.com/guardian/support-frontend/blob/5ddfca4d311ad85a3bfb3d32cba37353f0448246/support-lambdas/it-test-runner/src/main/scala/com/gu/RunITTests.scala#L40